annotation-processing/src/integrationTest/java/com/exactpro/epfast/CreatorTests.java

```diff
@@ -1,13 +1,17 @@
 package com.exactpro.epfast;
 
-import com.exactpro.epfast.annotation.internal.CreatorImpl;
+import com.exactpro.epfast.annotation.internal.packages.com.exactpro.epfast.inside.CreatorImpl;
+import com.exactpro.epfast.inside.DefAnn;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class CreatorTests {
-    private CreatorImpl creator = new CreatorImpl();
+    private com.exactpro.epfast.annotation.internal.packages.com.exactpro.CreatorImpl creator =
```

Comment:
You should use service to get creator instances
Our service helper should be able to get instance by package name (as specified in our annotation)